### DISCUSSION TITLE
Add note taking 'guidance' to the User Research section

### DISCUSSION THREAD

**Discussion by nhsbsa-sthil** [nhsbsa/nhsbsa-digital-playbook#217] [UCD]
## What

To help standardise the way we take notes in our user research sessions across the NHSBSA, we propose to create a number of best practice 'templates' for each User Researcher to use (see https://github.com/nhsbsa/nhsbsa-digital-playbook/discussions/216).

To ensure that the notes are as helpful as possible, we propose to create some 'guidance' for the note taker to follow.

## Why

Currently each User Researcher does things differently. This makes it difficult for each note taker to understand what is expected of them when they move between different projects / services.

Creating some clear 'guidance' on what is expected will help ensure that each User Researcher gets what they need.

**Comment by paigedoutel** [comment]
To add to **how** I think the guidance should be held in the playbook as best practice under [tools and resources](https://nhsbsa.github.io/nhsbsa-digital-playbook/research/research-tools-resources/) but maybe we need to change the name of it to tools, resources and best practice?

**Reply by paigedoutel** [reply]
sidenote if we do do this we may need to revisit #124 with @yantantether 's help as pages can become very long and messy very quickly, without clear distinction between tools, resources and best practice. One to discuss in the next working group meeting.

**Reply by aircwo** [reply]
Could be split into a display of articles? Cards in one place for Tools, Resources and Best Practice such like [this](https://nhsbsa.github.io/nhsbsa-digital-playbook/design/content-designer/).

**Comment by nhsbsa-deaco** [comment]
I was wondering if we could potentially include a new card as part of the User Research screen so along side, tools and resources, maybe having something like 'Supporting User Research' - 'How you can support user research'... opens up for other ideas of what support could be offered and all be logged in one place.

**Comment by lmilor** [comment]
[Writing notes DDaT guidance](https://nhsbsauk-my.sharepoint.com/:w:/g/personal/laumi_nhsbsa_nhs_uk/EeGgN3ZcSU1Eq-qrIRcoUxQBnoVFUssNZAXgva_kczEjEA?e=9E9QMy)

This has been reviewed by everyone in the working group and agreed. Next steps we could ask a UR and observer to look over them and see if they're fit for purpose. I have an assumption they're a bit long so would be good to explore further

**Reply by lmilor** [reply]
Jeannette and I are meeting with a UR next week to review this, will update :)

**Reply by nhsbsa-deaco** [reply]
I will take the as-is to make a start, then if there are any updates I can just tweak them 😄

**Comment by nhsbsa-deaco** [comment]
I'll take a look at the playbook and put some potential ideas together for how we are going to present this information and share anything here :)

**Reply by nhsbsa-deaco** [reply]
Here are 2 potential flows for the guidance, I have gone with a more generic flow since we could potentially start adding more to this.

1. This flow will go directly into the 'Supporting user research' page which could eventually get huge if we add more guidance

2. This flow will minimise the size of the screens if we do go ahead and add more guidance, each time we add more this will be its own post

Out of the 2 I personally prefer the second option just with the idea of guidance pages potentially getting very long, I also believe this would help scanning through the types of guidance we provide.

What's everyones thoughts? 😄 

![Dean Cobb - Communities - Observing and note-taking guidance](https://github.com/nhsbsa/nhsbsa-digital-playbook/assets/51478242/c290fe84-09fb-4736-bd31-98fc86d09761)

**Reply by nhsbsa-deaco** [reply]
Just to mention, this is the as-is guidance document shared in this discussion, the content is up for review and could change

**Reply by nhsbsa-sthil** [reply]
I'm inclined to go with Option 1 for now, but if / when further articles are needed around 'supporting UR' - it's probably at that point that you'd amend the structure to follow Option 2?

**Reply by nhsbsa-deaco** [reply]
yeah fair point 👍 I guess the navigation work will help direct this too

**Comment by nhsbsa-deaco** [comment]
New issue created for this work #229

**Comment by nhsbsa-deaco** [comment]
Are we happy to go with this flow for now and then if we start to add more guidance on supporting user research we can amend the navigation. 

![1  Observing and note-taking guidance](https://github.com/nhsbsa/nhsbsa-digital-playbook/assets/51478242/a465a412-6a45-484c-827f-edecfdc5c568)

**Reply by lmilor** [reply]
That looks great to me! :)

**Comment by nhsbsa-deaco** [comment]
Note taking guidance is still under review

**Comment by nhsbsa-deaco** [comment]
@lmilor whats the latest on the note taking guidance, is this something which can be added to the playbook now or is there more to be done with it?

**Reply by lmilor** [reply]
We've been waiting for feedback from HwHC as it was being trialled in this area - I can chase

**Reply by nhsbsa-deaco** [reply]
ah no thats fine no need to chase i just couldnt remember where it was at and if it was just waiting for someone to pick up to put live 😄

**Comment by nhsbsa-deaco** [comment]
This is now live - https://nhsbsa.github.io/nhsbsa-digital-playbook/research/supporting-ur/
